recipes/seekr2_openmm_plugin/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+CMAKE_FLAGS="-DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release -DSEEKR2_BUILD_OPENCL_LIB=OFF -DOPENMM_DIR=$PREFIX"
```

Comment:
There is a provided CMAKE_ARGS variable that you can expand here. It already contains some useful defaults.
```suggestion
CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release -DSEEKR2_BUILD_OPENCL_LIB=OFF -DOPENMM_DIR=$PREFIX"
```